packages/expo-location/CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 ### 🛠 Breaking changes
 
+- Add `scope` field in returned value to indicate whether background permissions are granted. Add `android.accuracy` field to determine whether `coarse` or `fine` location permission is granted.([#9446](https://github.com/expo/expo/pull/9446) by [@mczernek](https://github.com/mczernek))
```

Comment:
This is not sufficient IMO 🤔 
You've made two changes:
1) You moved `scope` into `UMPermissionsResponse` ⬅ that affects `expo-permissions` and I believe separate `CHANGELOG` entry should be there.
2) You added this `android.accuracy` thing only in `expo-location` and that should be stated here in this package as you've done it.

So please add separate missing `CHANGELOG` entry for `expo-permissions` 🙏

Reply:
We've had discussion with @tsapeta - scope was actually used only in Location permission. So, including this in `expo-permissions` changelog under `Breaking changes` would be in fact disinformative - would suggest that anyone using `expo-permissions` should be warned, while only those using `expo-location` should.